client/views/user_name.tpl

```diff
@@ -1 +1 @@
-<span role="button" class="user {{colorClass nick}}" data-name="{{nick}}">{{mode}}{{nick}}</span>
+<span role="button" class="user {{colorClass nick}} nick-{{nick}}" data-name="{{nick}}">{{mode}}{{nick}}</span>
```

Comment:
And this?

Reply:
Same as the other one.

Reply:
Same here, you can find it by `data-name`.